test/plugins/common.ts

```diff
@@ -15,6 +15,15 @@
  */
 'use strict';
 
+import * as mocha from 'mocha';
+declare global {
```

Comment:
Do mocha type definitions not provide this?

Reply:
It does for `it` in the global scope, but now the `it` property on the `global` object. I'm not entirely clear on why there's a distinction, but `tsc` complains if this definition isn't provided.